platforms/access-bridge/src/errors.ts

```diff
@@ -0,0 +1,205 @@
+import { ServerResponse } from 'node:http';
+
+import { JWError, JWErrorResponse } from '@jwp/ott-common/types/errors.js';
+
+import { RequestMethod } from './http.js';
+
+/**
+ * List of possible error codes based on RFC-JW03-Delivery API
+ * https://www.notion.so/jwplayer/RFC-JW03-Delivery-API-fc35c5f5d300445eba4004de970d06b4
+ */
+export enum ErrorCode {
```

Comment:
I wonder if these 3 type definitions (ErrorCode, ErrorDescription, and ErrorStatusCode) in addition to the const error instances at the bottom could all be combined into 1 typed object like you did with the Endpoint Handlers. It fees like the 4 pieces can safely be bundled together to keep the data here more compact and keep the codes, types, and descriptions all together that belong together.

Reply:
good suggestion! grouped them into one object, seems lot clearer now.